tinyblog/articles/2016-10-03-evening-social.md

```diff
@@ -0,0 +1,41 @@
+# Evening social sponsored by Sentry!
+
+> Published on 3rd Oct 2016 by Ola Sitarska
+
+**This year's evening social event is brought to you by [Sentry](https://sentry.io/), a real-time crash
+reporting that gives your team insight into errors affecting your customers.**
+
+---
+
+We're excited to announce that the social evening this year will take place
+at Pllek, a very unique location in Amsterdam. We can't wait to meet you for an 
+evening of chatting and meeting each other!
+
+![](http://www.horecasupport.nl/admin_assets/hotspots/hotspots_slides/237-1-slide.jpg)
+
+Selection of light snacks, soft drinks, wine and beer will be provided courtesy
+of our sponsor [Sentry](https://sentry.io/). Huge thanks to them! 💕
+
+**Date and time:**  
+Friday, 4th of November, 8.30pm - 1am. 
+
+**Location:**  
+Pllek  
+TT Neveritaweg 59  
+1033 WB Amsterdam  
+[pllek.nl](http://www.pllek.nl/contact.html)
+
+**Getting there:**  
+Pllek is available either through public bus transportation or ferry. Detailed
+instructions can be found [here](https://goo.gl/maps/isuzG61hLJy).
+
+// TODO: @erik, can we provide some info here about getting back and when the 
```

Comment:
To get to Pllek, take a ferry to NDSM. Ferry 905 to NDSM leaves from behind Central Station, and departs every 30 minutes, at :15 and :45. Between 17:00 and 18:30 it leaves every 15 minutes. Ensure you take the right ferry, as they leave for other destinations too. The last ferry towards NDSM leaves Central Station at 23:45. From the ferry port, you can [walk to Pllek](https://www.google.nl/maps/dir/Amsterdam,+Veer+NDSM+Werf,+Amsterdam/Pllek,+Amsterdam/) in a few minutes.

The trip to NDSM takes about 15 minutes and has nice views - as you get close to NDSM you'll be able to see the former Amsterdam shipbuilding yard, Pllek itself on the side of the water, a hotel on a boat and a defunct Soviet submarine. There are some restaurants on the NDSM side, including [IJkantine](http://www.ijkantine.nl/en/home-2/) and [Loetje aan 't IJ](http://loetjeaantij.loetje.com).

Another option is to take ferry 903 from Westerdoksdijk, but this is harder to reach by transit. Note that NDSM may not be the first stop of this ferry. There are also buses going to NDSM.

To get back, you can take 906 ferry to Centraal Station at :00 and :30. The last ferry on this line leaves at 23:30, but there is a night ferry. Night ferry 904 leaves NDSM at 00:04, 00:49, 01:34 and a few later times. Ferry 904 goes to Westerdoksdijk and then Central Station, so make sure you get off at the right port.